pylearn2/datasets/csv_dataset.py

```diff
@@ -119,12 +128,12 @@ def __init__(self,
 
         # and go
         self.path = preprocess(self.path)
-        X, y = self._load_data()
+        x, y = self._load_data()
 
         if self.task == 'regression':
-            super(CSVDataset, self).__init__(X=X, y=y, **kwargs)
+            super(CSVDataset, self).__init__(X=x, y=y, **kwargs)
         else:
-            super(CSVDataset, self).__init__(X=X, y=y,
+            super(CSVDataset, self).__init__(X=x, y=y,
                                              y_labels=np.max(y) + 1, **kwargs)
```

Comment:
OK, thanks for the explanation.
I think single-letter uppercase is not an issue at least for the pep8 script we are using.
It is not necessary that you revert everything back to capital X, but I think it would avoid some confusion.